### PR TITLE
Update docker section for ssh access

### PR DESCRIPTION
**Pull request by gaelreyrol**
A little trick that helps a lot in case you're stuck with a docker container !

### CODE REVIEW

**Comment by ldoguin** [comment]
@gaelreyrol  I would replace 

> On the other hand,

with *instead*.

**Comment by gaelreyrol** [comment]
@ldoguin Done !

File: administrate/ssh-clever-tools.md [hunk 1]
```diff
@@ -24,7 +24,9 @@ While direct SSH access to instances is not recommended in an [immutable infrast
 
 {{< alert "warning" "SSH access is disabled on Docker instances" >}}
     <p>
-    For security reasons, SSH access is disabled for Docker applications. Contact the support if you need help to debug your Docker applications.
+    For security reasons, you can't connect on the instance that hosts your Docker application.<br/>
+    Instead, if bash is installed in your Docker application, you will be able to connect directly inside it.<br/>
```
Comment (BlackYoup) [inline administrate/ssh-clever-tools.md:28]:
```suggestion
    Instead, if `bash` is installed in your Docker application, you will be able to connect directly inside it.<br/>
```

Comment (BlackYoup) [reply administrate/ssh-clever-tools.md:28]: Might as well do `s/inside it/inside your container/`

**Comment by BlackYoup** [comment]
Thanks!

**Pull request merged**